examples/mockups/index.html

```diff
@@ -57,7 +57,7 @@ <h1 class="ui header massive">
               </div>
               
               <div>
-                <a href="https://docs.readthedocs.io/page/intro/getting-started-with-sphinx.html" class="ui button large secondary gradient fade">Start your docs</a>
+                <a href="https://docs.readthedocs.io/page/intro/getting-started-with-sphinx.html" class="ui button large secondary gradient fade">Import your docs</a>
```

Comment:
"Start your docs" sounded weird, hope this sounds better? And also signals that the first action the user will take is "importing" their docs, as per our current UI

Reply:
Love it!

Reply:
"Get started!" is simple, to the point and works. "Import your docs" had the +1 of telling you something about the process.

By this logic, after reading the issue you linked I would also consider "Add your project" or "Connect your repo". Only, these might not sound as clear as "Get started!"

Reply:
True. Though I just realized that if we want to link to the Sphinx getting started guide, this doesn't align with any call to action that references add an existing documentation project. The getting started guide walks the users through starting a project from scratch, whereas a call to action like "Add your project" is engaging a user that already has a Sphinx project configured.

So I suppose I would vote to keep this pointing to the Sphinx guide, and make the language in the link something like "Get started!" or maybe even language more specific to starting a project from scratch. Targeting this user here is probably the most helpful, as a user familiar with Sphinx already probably doesn't need to be sold on RTD as much.

Reply:
Hah yeah, I had the same thought on two buttons. This could be something to test later for sure.

For now, sounds like we agree starting at `Get started!` and can tweak this a little bit. Having a link into our Sphinx guide is a great benefit here though, that doc does a great job of highlighting what the user should expect.

```suggestion
                <a href="https://docs.readthedocs.io/page/intro/getting-started-with-sphinx.html" class="ui button large secondary gradient fade">Get started!</a>
```